### DISCUSSION TITLE
permissions for different proyects

### DISCUSSION THREAD

**Discussion by hernanPSS** [kimai/kimai#4719] [Questions & Answers]
I want to split the permissions for 1 user to be a lider (or other role)  in one project/customer and a different one in another project of other client.... is that possible?

**Comment by hernanPSS** [comment]
the problem I find is that I have some "Administrative Projects", like "Holidays" and I don´t want to see the leaders of a project to view the holidays hours but they have to use thay project for their own holidays....

**Reply by kevinpapst** [reply]
Holidays can be managed with the https://www.kimai.org/store/controlling.html plugin

**Comment by hernanPSS** [comment]
it is not just about holidays.... it is about all projects... I'm not been able to set a normal user role for a user in one project and a more advanced role in other... :( 
if I set a custom role for a user it seems it is for all projects that the user is member

**Reply by kevinpapst** [reply]
A user can ones the assigned roles (that is teamlead as it sounds in your case) globally, limiting his functional permissions.

A user can also be either a member or a teamlead of a team, limiting the visibility of users. 
A user can have different positions in different teams. 
A member cannot see the data of other team member. 
A teamlead can see the data of his team members. 

This permission setup is limited, but enough for > 80% of all use-cases.
